client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -126,7 +126,7 @@
          * Autosave an item
          */
         this.save = function saveAutosave(item) {
-            if (item._editable && item._locked) {
+            if (item._editable && !item._locked) {
```

Comment:
That's not always true. There are some watches on `item` which sets field values using javascript. Once the values are set the `autosave.save()` is triggered even though the item isn't opened for edit. So this check prevents autosaving if the item isn't opened for edit and isn't locked by other users.